sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -153,11 +153,12 @@ object CommandUtils extends Logging {
    * for each path.
    * @param sparkSession the [[SparkSession]]
    * @param paths the Seq of [[Option[Path]]]s
-   * @return total size of all partitions
+   * @return a Seq of same size as `paths` where i-th element is total size of `paths(i)` or 0
```

Comment:
same size?

Reply:
the returned `Seq`.size is equal to `paths`.size.